python/pip_install/requirements.bzl

```diff
@@ -76,6 +76,9 @@ def compile_pip_requirements(
         requirement("pip_tools"),
         requirement("setuptools"),
         requirement("tomli"),
+        requirement("importlib_metadata"),
+        requirement("zipp"),
+        requirement("more_itertools"),
```

Comment:
Are these all noops in higher language version? I know `import_metadata` is. I'm surprised this still works with versions like 3.6 given the version of pip-tools used here is now incompatible with that version.

Reply:
I can't speak for 3.6, but this rule definitely doesn't work for 3.7, which is the version we're trying to work with. Far as I can see, the other two aren't nops, but they should be compatible with the relevant interfaces.

Reply:
Yeah I guess we've dropped 3.6 support here given it's EoL, so that's fine. Thanks for checking!